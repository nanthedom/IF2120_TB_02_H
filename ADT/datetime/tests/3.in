3
30
1
2004
2
2
2