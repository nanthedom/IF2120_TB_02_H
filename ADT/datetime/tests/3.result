ADT/datetime/tests/3.in ADT/datetime/tests/3.out: TRUE
