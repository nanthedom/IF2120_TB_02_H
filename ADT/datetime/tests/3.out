30/01/2004 02:02:02